### ISSUE TITLE
[DEMO]

### ISSUE THREAD

**Issue by emirhanai** [PennyLaneAI/qml#636]
#### General information

**Name**
Emirhan BULUT

**Affiliation** (optional)
Individual

**Twitter** (optional)
LinkedIn: https://www.linkedin.com/in/artificialintelligencebulut/

**Image** (optional)
https://camo.githubusercontent.com/b317dbd696c79f7949e50eb52741fc51c9ee168555bdd77cf6432906f5903454/68747470733a2f2f7261772e67697468756275736572636f6e74656e742e636f6d2f656d697268616e61692f534f322d456d697373696f6e2d50726564696374696f6e2d66726f6d2d44696573656c2d456e67696e65732d776974682d5175616e74756d2d546563686e6f6c6f67792d35472f6d61696e2f534f32253230456d697373696f6e25323050726564696374696f6e25323066726f6d25323044696573656c253230456e67696e6573253230776974682532305175616e74756d253230546563686e6f6c6f6779253230283547292e706e67

--------------------------------------------------------------------------------

#### Demo information

**Title**
SO2 Emission Prediction from Diesel Engines with Quantum Technology (5G)

**Abstract**
A worldwide study has been conducted on the emission values of SO2 gases released from diesel engines in the world (class 1 if it has increased compared to the previous year, class 0 if there has been a decrease compared to the previous year, and class 0 for the starting years). In this research, 5G compatible quantum algorithms were designed by me. Quantum computer was used for the process. The minimum number of qubits is set for use on all computers. Finally, the same data was tested in the classical deep neural network (deep learning) network and Machine Learning algorithm (Random Forest). On the basis of test accuracy, the quantum5 algorithm was found to be more performant than all of them.

**Relevant links**
https://github.com/emirhanai/SO2-Emission-Prediction-from-Diesel-Engines-with-Quantum-Technology-5G

**Comment by KetpuntoG** [comment]
Thanks for open the issue! It looks like a very interesting project. It would be great if you could add some text explaining the thread of what you are doing 😄 The model looks interesting and it would be great to visualize it! Even if it is a hand drawn drawing

**Comment by emirhanai** [comment]
> Thanks for open the issue! It looks like a very interesting project. It would be great if you could add some text explaining the thread of what you are doing 😄 The model looks interesting and it would be great to visualize it! Even if it is a hand drawn drawing

Hey KetpuntoG!
I added it to readme.MD at the now.

### Aim

The aim of the project is to design a Quantum Artificial Intelligence brain that learns the emission values ​​of SO2 gases released from diesel engines in the world with the 8-layer Quantum Algorithm, and then to compare the performance of the created Quantum Artificial Intelligence brain with Machine Learning, Deep Learning (Classical Neuronal Networks).

### Performance (Accuracy)

Quantum Artificial Intelligence algorithms have been proven to be more performant than Machine Learning and Deep Learning artificial intelligence systems.

### Speed

Quantum AI algorithms have been proven to be faster than Machine Learning and Deep Learning artificial intelligence systems.

### Energy

It has been proven that Quantum Artificial Intelligence algorithms use less energy in commercial/academic uses after model formation than Machine Learning and Deep Learning artificial intelligence systems. You can access this proof by file size.

### Usage Area

This project has proven to be compatible with 5G technology. The result obtained from the logarithm of the division of the byte rates transferred at 5G speed to the model accuracy is more than the result obtained in 4G technology, as in the following mathematical calculation.

5G = 10Gbps

4G = 0.1Gbps

**log(5G/model_byte) = 2.698970004336X**

**log(4G/model_byte) = 0.69897000433602X**

I proudly present this software,

Thank you.

Emirhan BULUT

**Comment by emirhanai** [comment]
> Thanks for open the issue! It looks like a very interesting project. It would be great if you could add some text explaining the thread of what you are doing 😄 The model looks interesting and it would be great to visualize it! Even if it is a hand drawn drawing

And I create and Added Model Chart (I added it to  Readme.MD):

https://raw.githubusercontent.com/emirhanai/SO2-Emission-Prediction-from-Diesel-Engines-with-Quantum-Technology-5G/main/keras_my_model_chart.png

**Comment by KetpuntoG** [comment]
We usually put the twitter user only but I'll talk to marketing to see what can be done.
 I'll take care of introducing it on the web. Thanks for the work done @emirhanai !

**Comment by emirhanai** [comment]
> We usually put the twitter user only but I'll talk to marketing to see what can be done. I'll take care of introducing it on the web. Thanks for the work done @emirhanai !

Thank you. I eagerly await its release.

**Comment by KetpuntoG** [comment]
At the end we will put the twitter username but you can always comment putting the link to the linkdn if you want :) Tomorrow we will make the publication. Thanks again!

**Comment by emirhanai** [comment]
> At the end we will put the twitter username but you can always comment putting the link to the linkdn if you want :) Tomorrow we will make the publication. Thanks again!

Thank you 🙏🏻

**Comment by emirhanai** [comment]
> At the end we will put the twitter username but you can always comment putting the link to the linkdn if you want :) Tomorrow we will make the publication. Thanks again!

Hello, will it be prepared to be published on your website as a page? Thank you

**Comment by KetpuntoG** [comment]
Hello @emirhanai , you can see it here:
https://pennylane.ai/qml/demos_community.html

**Comment by emirhanai** [comment]
> Hello @emirhanai , you can see it here: https://pennylane.ai/qml/demos_community.html

Yes i know but we can share like tutorial. What about?

**Comment by KetpuntoG** [comment]
The work done is more suitable for community demos. In this case we will not perform a demo.
The community demos section is a place of interest to share your projects in pennylane, I am sure it will reach many people 🙂
